packages/controllers/src/utils/ConnectorUtil.ts

```diff
@@ -181,6 +181,38 @@ export const ConnectorUtil = {
       return 0
     })
   },
+
+  /**
+   * Returns the priority of a connector.
+   * @param connector - The connector to get the priority of.
+   * @returns The priority of the connector.
+   */
+  getPriority(connector: ConnectorWithProviders) {
```

Comment:
Is this 'priority' only for Coinbase/Base wallets? If so can you add description for it? It's not understandable why we have this Priority concept needed